doc/readme_link.rst

```diff
@@ -29,8 +29,8 @@
 
     fig, axs = plt.subplots(1, 2, figsize=(6, 3))
 
-    axs[0].imshow(forward_abel, cmap='ocean_r')
-    axs[1].imshow(inverse_abel, cmap='ocean_r')
+    axs[0].imshow(forward_abel, clim=(0, None), cmap='ocean_r')
```

Comment:
Oh, I didn't realize that you could pass `None` to the clim arg! I'm going to use this!! :)

Reply:
It seems that using `None` for “default” is a general convention. Better check the documentation, though... :–)